### DISCUSSION TITLE
Is there a way to keep the previous time when use zoom？

### DISCUSSION THREAD

**Discussion by Idletalk** [katspaugh/wavesurfer.js#3101] [Q&A] [answered]
When I scroll the container to a certain point（such as 5s in visible center）, then  I reset zoom,  the render will scroll to 0. 

![img_v2_91afe473-7407-42ab-8c22-baf6c6046fag](https://github.com/katspaugh/wavesurfer.js/assets/38195804/fbf27f04-9773-4ba9-84a2-ad9426aff599)
Is there any way I can set the zoom value ，after this action the point 5s  is also in my windowing center？

**Comment by Idletalk** [comment]
I see the code when the zoom "reRender", it will  clear canvases
![image](https://github.com/katspaugh/wavesurfer.js/assets/38195804/e9cdca4f-8919-4669-a48a-eeb645b7f725)
and then the "scrollLeft" will be 0
![image](https://github.com/katspaugh/wavesurfer.js/assets/38195804/57190ac2-2dfb-4486-99d5-52fe021b8bf2)

**Comment by katspaugh** [comment, marked as answer]
The current logic tries to always keep the playhead in the same place when zooming. E.g. if you click on that 5 sec mark, it will stay there.
I think the behavior you expect also makes sense and this can be done. I've created an [issue](https://github.com/katspaugh/wavesurfer.js/issues/3102) for it.
